fili-core/src/test/groovy/com/yahoo/bard/webservice/druid/model/response/DruidResponseSerializationSpec.groovy

```diff
@@ -0,0 +1,61 @@
+package com.yahoo.bard.webservice.druid.model.response
+
+import static com.yahoo.bard.webservice.druid.model.response.GroupByResultRow.Version.V1
+import static com.yahoo.bard.webservice.druid.model.response.GroupByResultRow.Version.V2
+import static java.util.Arrays.asList
+
+import com.fasterxml.jackson.databind.ObjectMapper
+
+import org.joda.time.DateTime
+
+import spock.lang.Specification
+import spock.lang.Unroll
+
+class DruidResponseSerializationSpec extends Specification {
+    static DateTime DATE_TIME = DateTime.parse("2017-06-13T00:00:00.000Z");
+    static ObjectMapper objectMapper = new ObjectMapper();
+    static key1 = "key1"
+    static key2 = "key2"
+
+    @Unroll
+    def "Check Timeseries druid response serialization"() {
```

Comment:
If Unrolls it, it's common to add a description for each different test case. For example https://github.com/yahoo/fili/blob/master/fili-core/src/test/groovy/com/yahoo/bard/webservice/table/availability/MetricUnionAvailabilitySpec.groovy#L90